modules/call-dispatch/src/lib.rs

```diff
@@ -147,6 +234,39 @@ impl<T: Trait> MessageDispatch<T::MessageId> for Module<T> {
 	}
 }
 
+/// Verify payload of the message at the sending side.
+pub fn verify_sending_message<
```

Comment:
A call to this function should be added to some kind of payload verification in the message delivery pallet, right?
I mean that the message delivery upon `add_message` call should delegate the payload verification to the dispatcher, and that's where we'd call this function?
I guess we don't have a trait for that and it's not really plugged into the runtime, so that's why it's not used apart from the tests?

Reply:
Yes - it should be verified during `send_message()` call. That's what's `LaneMessageVerifier` from #339 is for